### DISCUSSION TITLE
keep array reactivity

### DISCUSSION THREAD

**Discussion by nablaFox** [vuejs/vuefire#1286] [Questions and Help] [answered]
I have a firestore database structured as follows:

```
rooms (collection)
	room1 (document)
		name
		members [
			user1,
			user2
		]
		
```
how can i make currentRoom:

`const currentRoom = useDocument(doc(collection(db, 'rooms'), roomId))`

responsive to changes in the members array?

**Comment by posva** [comment, marked as answer]
with a computed, like in vue:

```ts
const currentRome = useDocument(computed(() => doc(collection(db, 'room's), roomId.value)))
const currentRome = useDocument(computed(() => doc(collection(db, 'room's), route.params.roomId)))
```

It needs to be a reactive value though, like a ref or a route param
